sections/advanced.md

```diff
@@ -742,6 +742,9 @@ my_job:
       - ${PRE_COMMIT_HOME}
 ```
 
+pre-commit's cache requires to be served from a constant location between the different builds. This isn't the default when using k8s runners
+on Gitlab. In case you face the error `InvalidManifestError`, set `builds_dir` to something static e.g `builds_dir = "/builds"` in your `[[runner]]` config
```

Comment:
while I don't care personally, someone else will send me another PR to change this to "GitLab" so it's probably best to do this here first!